File: Java/Lista 3/L3_Ex1.3/src/Estrela.java

public class Estrela {
    
    private String nome;
    private Double temperatura;
    private String cor;

    public Estrela(String nome, Double temperatura, String cor) {
        this.nome = nome;
        this.temperatura = temperatura;
        this.cor = cor;
    }
    
    public void mostar(){
        System.out.println("Nome: " + this.nome);
        System.out.println("Temperatura: " + this.temperatura+"C");
        System.out.println("Cor: " + this.cor);    
    } 

    public Double getTemperatura() {
        return temperatura;
    }
     
}

File: Swift/Lista_03_EX03/Estrela.swift
import Foundation
 
 public class Estrela{
 
    private var nommeEstrela : String = ""
	private var cor : String = " "
	private var temperatura : Double
	
     
    init(nomeEstrela : String, cor : String, temperatura : Double ) {
        self.nomeEstrela = nomeEstrela
		self.cor = cor
		self.temperatura = temperatura
    }
  
	func mostrar( ) {
		print("Nome: \(nomeEstrela)")
		print("Temperatura: \(temperatura)")
		print("Cor: \(cor)")
	}
	
	public func getTemperatura() -> Double{
      return temperatura
    }
	
}